packages/desktop-client/src/components/accounts/AccountSyncCheck.js

```diff
@@ -31,7 +31,7 @@ function getErrorMessage(type, code) {
     case 'API_ERROR':
       switch (code.toUpperCase()) {
         case 'PLANNED_MAINTENANCE':
-          return 'Our servers are currently undergoing maintenance and will be available again soon.';
```

Comment:
Unrelated to your change: I think we can clean up the `PLANNED_MAINTENANCE` code paths. They are deprecated.